vitrina/projects/forms.py

```diff
@@ -12,6 +13,11 @@ class ProjectForm(ModelForm):
     description = CharField(label=_('Aprašymas'), widget=Textarea)
     url = CharField(label=_("Nuoroda į panaudojimo atvejį"), required=False)
     image = ImageField(label=_("Paveiksliukas"), required=False)
+    dataset = ModelMultipleChoiceField(
+        label=_('Duomenų rinkiniai'),
+        queryset=Dataset.objects.all(),
+        required=False
+    )
```

Comment:
Do not show this field, because there are too many datasets to select them this way, via a field. Datasets and projects should be associated from dataset details page, see https://github.com/atviriduomenys/katalogas/issues/239.